whitepapers/mechanisms/analytic_gaussian/analytic_gaussian.tex

```diff
@@ -0,0 +1,46 @@
+\documentclass[11pt]{scrartcl} % Font size
```

Comment:
This tex file is completely commented out? Should we maybe just delete it from develop for now?

Reply:
Oops, yes. Didn't mean to push that. I'll remove it.